docs/api/sandbox-option.md

```diff
@@ -162,16 +162,16 @@ feature. We are still not aware of the security implications of exposing some
 Electron renderer APIs to the preload script, but here are some things to
 consider before rendering untrusted content:
 
-- A preload script can accidentally leak privileged APIs to untrusted code.
+- A preload script can accidentally leak privileged APIs to untrusted code,
+  unless [`contextIsolation`](../tutorial/security.md#3-enable-context-isolation-for-remote-content)
+  is also enabled.
 - Some bug in V8 engine may allow malicious code to access the renderer preload
   APIs, effectively granting full access to the system through the `remote`
-  module.
+  module. Therefore, it is highly recommented to either
+  [disable](../tutorial/security.md#15-disable-the-remote-module) or
+  [filter](../tutorial/security.md#16-filter-the-remote-module) the `remote` module.
```

Comment:
Maybe make it clear here that filtering should be considered only if disabling isn't possible. Something like "filter the `remote` module, if disabling is not feasible." I know that this is stated in the security tutorial but  it would cost only a few more words to advance it here.

Reply:
what about now?

Reply:
Great!